lib/public/Migration/Attributes/TableMigrationAttribute.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+namespace OCP\Migration\Attributes;
+
+use JsonSerializable;
+
+class TableMigrationAttribute extends MigrationAttribute implements JsonSerializable {
+	public function __construct(
+		string $table = '',
```

Comment:
```suggestion
		string $table,
```

table migrations without a name are impossible. the table name has to be mandatory

Reply:
should be make it mandatory for any migration attribute ?

Reply:
Yes